lib/components/flare_component.dart

```diff
@@ -33,6 +30,13 @@ class FlareComponent extends PositionComponent {
     }
   }
 
+  void updateSize(Vector2 newSize) {
```

Comment:
can't we overide a custom setter to the size property as it was before with (width, height)?

Reply:
Fixed!